js/index.js

```diff
@@ -1 +1,112 @@
-// Your code goes here
+// Variables
+const mainNav = document.querySelector('.main-navigation');
+const funBusMain = document.querySelector('.logo-heading');
+const nav = document.querySelectorAll('.nav-link');
+const footer = document.querySelector('.footer');
+const imgs = document.querySelectorAll("img");
+const body = document.querySelector("body");
+const button = document.querySelectorAll('.btn');
+const anchorText = document.querySelectorAll('a');
+const paragraphText = document.querySelectorAll('p');
+const h1 = document.querySelector('h1');
+const h2 = document.querySelectorAll('h2');
+const h4 = document.querySelectorAll('h4');
+
+
+/*===================Event listeners===================*/
+
+//click on fun bus logo, changes color
+funBusMain.addEventListener('click', () =>{
+    funBusMain.style.color = 'darkkhaki';
+});
+
+//Nav elements mouseover Grow and Color Change
+for (let i = 0; i < nav.length; i++){
+    nav[i].addEventListener('mouseover', () =>{
+        nav[i].style.fontSize = '1.65rem';
+        nav[i].style.color = 'lightseagreen';
+    });
+}
+//Nav elements shrink back and color change
+for (let i = 0; i < nav.length; i++){
+    nav[i].addEventListener('mouseout', (event) =>{
+        nav[i].style.fontSize = '1.6rem';
+        nav[i].style.color = 'black';
+    });
+}
+
+//mouse over nav makes it blue
+mainNav.addEventListener('mouseover', (event) => {  
+    event.preventDefault();
```

Comment:
if there isn't a default for mouseover (I don't think there is in this case), you don't need to use event.preventDefault